### PR TITLE
Adds glitch weather API to take home interview weather project.

### LINKED ISSUE #15598: Fix the "Show the Local Weather" project
#### Challenge Name
[Show the local weather](https://www.freecodecamp.com/challenges/show-the-local-weather)

#### Issue Description
Currently this project has at least two major problems:
1) It recommends more than one API. We should just have a single API that reliably works on CodePen
2) It suggests putting private keys in a visible place on Codepen. This is an anti-pattern.

Here's what I propose we do. We build a Glitch.com app that serves as an API. The Glitch App can then reach out to the Dark Sky (or whatever API works best) on the user's behalf and pass back the data.

This way, the keys are obscured, and the user has a simplified API they can interact with. There's no need for them to shop around for APIs or worry about security issues with CodePen.

We have already done something like this with our [Twitch.tv challenge](https://www.freecodecamp.org/challenges/use-the-twitchtv-json-api).

### PR DESCRIPTION
**Pull request by MiloATH**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue: Closes #15598

#### Description

This will add the glitch weather API to the weather project challenge in the take home interview section. This means that users can get the weather without using an API key and using a HTTP Secure connection.

### CODE REVIEW

File: seed/challenges/08-coding-interview-questions-and-take-home-assignments/take-home-interview-projects.json [hunk 1]
```diff
@@ -13,8 +13,7 @@
...
-        "Either weather API service will require creating a free API key. Normally you want to avoid exposing API keys on CodePen, but we haven't been able to find a keyless API for weather.",
+        "<strong>Note:</strong> Many internet browsers now require an HTTP Secure (<code>https://</code>) connection to obtain a user's locale via HTML5 Geolocation. For this reason, we recommend using HTML5 Geolocation to get user location and then use the FreeCodeCamp Weather API <a href='https://fcc-weather-api.glitch.me' target='_blank'>https://fcc-weather-api.glitch.me</a> which uses an HTTP Secure connection for the weather. Also, be sure to connect to <a href='https://codepen.io' target='_blank'>CodePen.io</a> via <code>https://</code>.",
```
Comment (raisedadead) [inline seed/challenges/08-coding-interview-questions-and-take-home-assignments/take-home-interview-projects.json]:
```diff
- ...FreeCodeCamp
+ ...freeCodeCamp
```

Please update the styling.

**Review by raisedadead — CHANGES_REQUESTED** [review]

File: seed/challenges/08-coding-interview-questions-and-take-home-assignments/take-home-interview-projects.json (hunk 1 above)
Comment (MiloATH) [inline seed/challenges/08-coding-interview-questions-and-take-home-assignments/take-home-interview-projects.json]: Thanks. I just updated the styling.

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
Thanks @MiloATH

**Comment by QuincyLarson** [comment]
@MiloATH Wow - excellent work! Thank you for fixing this! Could you also make this change off backup/master and create a pull request there? We should get this into production ASAP. Once it's merged there, @BerkeleyTrue can deploy it.

**Pull request merged**